packages/odyssey-storybook/src/guidelines/DownstreamDevelopment.stories.mdx

```diff
@@ -0,0 +1,313 @@
+  },
+};
+```
+
+[2]: https://webpack.js.org
+
+### Browserslist
+
+Begin by creating a [`.browserslistrc` config file][3] at the root of your project:
+
+```
+extends @okta/browserslist-config-odyssey
+```
+
+Alternatively, add a `browserslist` entry to your package.json file.
+
+This config will allow postcss, babel and other toolchains to optimize
+your build based on the intended browser targets. The configuration provided
+from the `@okta/browserslist-config-odyssey` package mirrors
+[Okta's own published browser support list][4]. You should feel free to
+manipulate this configuration further to optimize your asset delivery.
+For example, to construct separate modern and IE11 only bundles that
+include appropriate polyfills and transpiled source code.
+
+[3]: https://github.com/browserslist/browserslist
+[4]: https://help.okta.com/en/prod/Content/Topics/Miscellaneous/Platforms_Browser_OS_Support.htm
+
+### Babel
+
+Begin by creating a [`babel.config.js` config file][5] the root of your project:
+
+```js
+module.exports = {
+  presets: ["@okta/odyssey-babel-preset"],
+
+  env: {
+    test: {
+      presets: [
+        [
+          "@okta/odyssey-babel-preset",
+          {
+            odyssey: {
+              identityObjectProxy: true,
+            },
+          },
+        ],
+      ],
+    },
+    production: {
+      comments: false,
+    },
+  },
+};
+```
+
+You can further configure each part of this preset by passing arguments
+within your babel configuration. The preset accepts arguments for the
+following babel presets and plugins:
+
+- `@babel/preset-env`
+- `@babel/preset-react`
+- `@babel/preset-typescript`
+- `@babel/odyssey-babel-plugin`
+
+Keep in mind you may want to configure the preset depending on the target
+environment. For example, tests may be built differently than local
+development code which is built differently than production code. An
+example of this is shown in the configuration above which saves time
+in unit test environments by enabling the `identityObjectProxy` option.
+Test coverage for styles can then be provided at the integration and VRT layer.
+
+[5]: https://babeljs.io
+[6]: https//www.github.com/okta/odyssey
+
+### PostCSS
+
+Begin by creating a [`.postcssrc.js` config file][7] at the root of your project:
+
+```js
+const { default: postcssOdyssey } = require("@okta/odyssey-postcss-preset");
+
+module.exports = (ctx) => {
+  if (!ctx.odyssey) {
+    return {};
+  }
+
+  const options = Object.assign(
+    ctx.odyssey,
+    ctx.env === "production"
+      ? {
+          logical: {
+            dir: "ltr",
+            preserve: false,
+          },
+          autoprefixer: {
+            grid: "autoplace",
+            env: "production",
+          },
+        }
+      : {
+          modules: {
+            ...ctx.odyssey.modules,
+            generateScopedName: "[name]__[local]___[hash:base62:6]",
+          },
+        }
+  );
+
+  return {
+    plugins: [postcssOdyssey(options)],
+  };
+};
```

Comment:
If possible, I'd recommend renaming `ctx` to `context`. It took me years until I knew what `ctx` meant.

In general, I prefer verbose variable names to make things as clear as possible. Minifiers will always fix that for you, but this is server-side only anyway.